src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -450,21 +450,13 @@ export function getExpandedResults(
       return;
     }
 
-    // otherwise just use exploded.field as a column
-
-    if (!exploded.field) {
-      // edge case: transform count() into id
-
-      if (exploded.aggregation !== 'count') {
-        fieldsToDelete.push(indexToUpdate);
-        return;
-      }
-
+    // edge case: transform count() into id
+    if (exploded.aggregation === 'count') {
       exploded.field = 'id';
     }
 
-    if (transformedFields.has(exploded.field)) {
-      // this field is duplicated in another column. we remove this column
+    if (!exploded.field || transformedFields.has(exploded.field)) {
```

Comment:
It looks like previously, if `exploded.field` was falsey, it was given a value of `id`, is that not needed?

Reply:
I don't think so. Other aggregate aliases, and functions should unpack into real columns based on the `TRANSFORM_AGGREGATES` mapping. If they don't we can drop those fields.

Reply:
I wrote that in a weird way.